astropy/cosmology/flrw/base.py

```diff
@@ -49,12 +50,12 @@ def quad(*args, **kwargs):
 _H0units_to_invs = (u.km / (u.s * u.Mpc)).to(1.0 / u.s)
 _sec_to_Gyr = u.s.to(u.Gyr)
 # const in critical density in cgs units (g cm^-3)
-_critdens_const = (3 / (8 * pi * const.G)).cgs.value
+_critdens_const = (3 / (8 * pi * cconst.G)).cgs.value
```

Comment:
If you're going to cgs anyway, is there a point in changing this?

Reply:
Not much, except for internal consistency. When the API covers all the necessary constants, I'd like to only be importing `astropy.cosmology.constants`